core/common/src/main/java/alluxio/underfs/options/CreateOptions.java

```diff
@@ -24,6 +24,9 @@
 @PublicApi
 @NotThreadSafe
 public final class CreateOptions {
+  // Determine whether to create any necessary but nonexistent parent directories.
```

Comment:
can you please add a comment to state the guideline when a caller should set to this `true`?